src/views/Pools/components/CakeVaultCard/VaultStakeModal.tsx

```diff
@@ -174,7 +175,7 @@ const VaultStakeModal: React.FC<VaultStakeModalProps> = ({ pool, stakingMax, isR
       <BalanceInput
         value={stakeAmount}
         onUserInput={handleStakeInputChange}
-        currencyValue={`~${usdValueStaked || 0} USD`}
+        currencyValue={cakePriceBusd.gt(0) && `~${usdValueStaked || 0} USD`}
```

Comment:
I think this is unnecessary since it is checked above already, it would show 0 USD even without this condition

Reply:
If we remove that check and If cakePriceBusd is NaN (when price is being loaded or something wrong when loaded) inside the model it will always show ~0 usd below the stake amount which can confuse the user.

Reply:
hm... makes sense 👍